src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/Pagination.scala

```diff
@@ -10,11 +11,11 @@ import doobie.implicits._
   * which seems wrong.
   */
 trait SQLPagination {
-  def limit[A: Read](lim: Int)(q: Query0[A]): Query0[A] =
-    (q.toFragment ++ fr"LIMIT $lim").query
+  def limit[A: Read](lim: Int)(q: Query0[A])(implicit pos: Pos): Query0[A] =
+    (Fragment(q.sql, Nil, Some(pos)) ++ fr"LIMIT $lim").query
```

Comment:
0.7.0-M3 dropped `.toFragment` on query, for some reason. This is essentially what that was doing in 0.6.0.